site/content/docs/5.2/components/breadcrumb.md

```diff
@@ -78,6 +78,13 @@ $breadcrumb-divider: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/
 
 You can also remove the divider setting `--bs-breadcrumb-divider: '';` (empty strings in CSS custom properties counts as a value), or setting the Sass variable to `$breadcrumb-divider: none;`.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This variant should not be used because it does not respect the Orange Design System specifications.
```

Comment:
We don't want to explain the reason, because the divider is mandatory?

Reply:
Yes, the divider is mandatory. => New proposition with this explanation:
> This variant **without the breadcrumb dividers** should not be used because it does not respect the Orange Design System specifications.
>
> Please refer to the [Local Headers/Breadcrumb](https://system.design.orange.com/0c1af118d/p/774477-local-headers/b/743cd0/i/66611057) guidelines on the Orange Design System website.